### PR TITLE
Task.11 - Create a page for courses that a student is enrolled in

### LINKED ISSUE #6: Task.11 - Create a page for courses that a student is enrolled in
When a student is logged in, they should be redirect to a page where all the courses they are enrolled in are shown. They should have a menu for enroll into a class using a class code

### PR DESCRIPTION
**Pull request by Kerryki**
When a student is logged in, they should be redirect to a page where all the courses they are enrolled in are shown. They should have a menu for enroll into a class using a class code

### CODE REVIEW

File: client/src/pages/CoursePage.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Link } from "react-router-dom"; // Import Link for internal navigation
+
+const CoursePage = () => {
```
Comment (Marian-Hristov) [inline client/src/pages/CoursePage.jsx]: Call this `StudentCourseView` it is more descriptive of who is actually viewing these courses. also add more functionality as to getting all the options of courses and displaying that to the user. They need a way to know what are the option of courses they can choose from

File: client/src/pages/CoursePage.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    padding: "10px 20px",
+    fontSize: "16px",
+    cursor: "pointer",
+    backgroundColor: "#007BFF",
+    color: "white",
+    border: "none",
+    borderRadius: "5px",
+    transition: "background-color 0.3s",
+  },
+};
```
Comment (Marian-Hristov) [inline client/src/pages/CoursePage.jsx]: please try to use tailwind css for these values! [https://tailwindcss.com/](Tailwind Search) you just put the css value in the search and it tells you how to format it for different values!

File: client/src/pages/CoursePage.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Link } from "react-router-dom"; // Import Link for internal navigation
+
+const CoursePage = () => {
+  const courses = [
+    { id: 1, name: "Course 1", imgSrc: "https://picsum.photos/id/237/200/301" },
+    { id: 2, name: "Course 2", imgSrc: "https://picsum.photos/200/300" },
+    { id: 3, name: "Course 3", imgSrc: "https://picsum.photos/id/257/200/300" },
+    { id: 4, name: "Course 4", imgSrc: "https://picsum.photos/id/238/200/300" },
+    { id: 5, name: "Course 5", imgSrc: "https://picsum.photos/id/239/200/300" },
```
Comment (Marian-Hristov) [inline client/src/pages/CoursePage.jsx]: Why is there pictures for the courses?

File: client/src/pages/CoursePage.jsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    // Add more courses as needed
+  ];
+
+  const handleAddCourse = () => {
+    alert("Add Course button clicked!");
+  };
+
```
Comment (Marian-Hristov) [inline client/src/pages/CoursePage.jsx]: Persistant users are not added yet, but you will eventually have to add logic for when you actually have a user or not.
If you don't have an user you redirect to student login, if you do have a user you just load the component normally

**Review by Marian-Hristov — COMMENTED** [review]
Minor tweaks here and there. I'll have further comments once I finish some changes on my side.

**Comment by Marian-Hristov** [comment]
So sorry! I hadn't seen #55 where most of the comments are addressed. I will review that now and leave comments

**Review by Vinuyans — APPROVED** [review]

**Pull request merged**